test/WebJobs.Extensions.CosmosDBCassandra.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.Tests.Common;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Security;
+using System.Reflection;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace Microsoft.Azure.WebJobs.Extensions.CosmosDBCassandra.Tests.Trigger
+{
+    public class CosmosDBTriggerAttributeBindingProviderTests
+    {
+
+        private readonly ILoggerFactory _loggerFactory = new LoggerFactory();
+        private static readonly IConfiguration _emptyConfig = new ConfigurationBuilder().Build();
+        private static CosmosDBCassandraOptions _options = new CosmosDBCassandraOptions();
+
+        public static IEnumerable<object[]> InvalidCosmosDBTriggerParameters
```

Comment:
You need to write scenarios that are valid for your new Trigger. You can take inspiration on the ones that already exist for the other triggers (Invalid parameters, parameters from AppSettings, parameters from Environment).